### ISSUE TITLE
Support for leptos 0.5

### ISSUE THREAD

**Issue by rambip** [eboody/styled#3]
In leptos 0.5, the `view` macro doesn't need `cx` anymore, which breaks your code.

**Comment by mendrik** [comment]
have the same problem

**Comment by dfeyer** [comment]
Some works has been done here: https://github.com/sauloco/styled/commit/589a3311174f23d3d33659b2723ab4488d43ca15

**Comment by ImTheSquid** [comment]
I switched support to 0.5 and added support for SSR. To use it just add the repo as a Cargo dep: https://github.com/ImTheSquid/styled, PR #4

**Comment by NiklasEi** [comment]
Nice, #4 was merged 👍 
The code examples in the readme still use the old Leptos context though. Is it still needed?

**Comment by ImTheSquid** [comment]
Nope, those need to be updated. Also the changes need to be mirrored/linked to the child READMEs since I forgot to update them.

**Comment by eboody** [comment]
i updated the crate to 0.3.2! can anyone help me check if everything works as expected?
